db/src/main/resources/db/migration/V1_006__add_claim_status_column.sql

```diff
@@ -0,0 +1,7 @@
+alter table claimant add column claim_status varchar(20);
+
+update claimant set claim_status = 'ACTIVE' where claim_status is null and eligibility_status = 'ELIGIBLE';
+
+update claimant set claim_status = 'REJECTED' where claim_status is null and eligibility_status != 'ELIGIBLE';
```

Comment:
this could actually just be `update claimant set claim_status = 'REJECTED' where claim_status is null;`

Reply:
Updated.